app/middleware/renderer.js

```diff
@@ -2,11 +2,11 @@ function results(req, res) {
   res.render('results');
 }
 
-function findHelp(req, res) {
-  res.render('find-help');
+function searchYourGP(req, res) {
```

Comment:
Why are we 'searching our GP'? Has he stolen some money :)

searchForYourGP might be better.

Also, probably need to be consistent with the camel casing for abbreviations (sometimes we use functionGP and sometimes functionGp). I think I prefer the former but recognise we are not consistent.